src/renderer/PresetsBar.tsx

```diff
@@ -20,16 +21,16 @@ const PresetListItem = ({
   handleDelete,
   isDisabled,
 }: IListItemProps) => {
-  const editValueRef = useRef<HTMLInputElement>(null);
+  const editModeRef = useRef<HTMLInputElement>(null);
```

Comment:
Can we undo these changes from `editValueRef` to `editModeRef`? This was a change I made in jat-90 based on @xenown's feedback